inference-engine/src/inference_engine/cnn_network_ngraph_impl.cpp

```diff
@@ -147,7 +147,7 @@ CNNNetworkNGraphImpl::CNNNetworkNGraphImpl(
     }
     for (auto& output : _outputData) {
         // Convert precision into native format. Be consistent with possible conversion to CNNNetwork later.
-        if (output.second->getPrecision() == Precision::I64) {
+        if (output.second->getPrecision() == Precision::I64 || output.second->getPrecision() == Precision::BOOL) {
```

Comment:
Without this change the tests failed with the following values error:
https://github.com/openvinotoolkit/openvino/pull/3669/checks?check_run_id=1575888630